tgui/packages/tgui/interfaces/PipeDispenser.js

```diff
@@ -0,0 +1,102 @@
+  const {
+    dispenser_ready,
+    windowName,
+    mobile,
+    removing_pipe,
+    laying_pipe,
+    max_disposal_pipes,
+  } = data;
+  return (
+    <Window
+      title={windowName}
+      width="300"
+      height={mobile ? 330 : 235}
+      fontFamily="Consolas"
+      font-size="10pt">
+      <Window.Content scrollable>
+        <Section>
+          {disposalpipes.map(disposalpipe => {
+            return (
+              <Flex key={disposalpipe.disposaltype} justify="space-between" align="stretch" style={{ "border-bottom": "1px #555 solid" }}>
+                <Flex.Item direction="row">
+                  {disposalpipe.image && (
+                    <Box style={{ "overflow": "show", "height": "32px" }}>
+                      <img
+                        src={`data:image/png;base64,${disposalpipe.image}`}
+                      />
+                    </Box>)}
+                </Flex.Item>
+                <Flex.Item direction="row"
+                  grow style={{
+                    "display": "flex",
+                    "justify-content": "center",
+                    "flex-direction": "column",
+                  }}>
+                  <Box>
+                    <Box inline>
+                      {disposalpipe.disposaltype}
+                    </Box>
+                  </Box>
+                </Flex.Item>
+                <Flex.Item direction="row"
+                  style={{
+                    "display": "flex",
+                    "justify-content": "center",
+                    "flex-direction": "column",
+                  }}>
+                  Amount:
+                  <NumberInput
+                    value={amount}
+                    minValue={1}
+                    maxValue={max_disposal_pipes}
+                    onChange={(e, value) => setAmount(Math.round(value))} />
+                </Flex.Item>
+                <Flex.Item bold direction="row" style={{
+                  "margin-left": "5px",
+                  "display": "flex",
+                  "justify-content": "center",
+                  "flex-direction": "column",
+                }}>
+                  <Button
+                    color={dispenser_ready ? "green" : "grey"}
+                    content="Dispense"
+                    disabled={!dispenser_ready}
+                    style={{ "width": "70px", "text-align": "center", "padding": "0px" }}
+                    onClick={() => act('dmake', { 'disposal_type': disposalpipe.disposaltype, 'amount': amount })}
+                  />
+                </Flex.Item>
+              </Flex>
+            );
+          })}
+        </Section>
+        {!!mobile && (
+          <Section
+            title="Automatic Pipe Options">
+            <Button
+              color={laying_pipe ? "average" : "green"}
+              content={laying_pipe ? "Stop Laying Pipe Automatically" : "Start Laying Pipe Automatically"}
+              fluid={1}
```

Comment:
```suggestion
              fluid
```
Should be equivalent unless I've forgotten something weird about how these work.

Reply:
Yeah I'll do that